README.md

```diff
@@ -44,121 +52,329 @@ $ man tar
-  Aggregates cheat sheets from multiple sources (including tldr-pages)
-  into 1 unified interface.
+📖 [Learn more about TL;DR](https://www.howtogeek.com/435266/what-does-tldr-mean-and-how-do-you-use-it/)
 
-- [devhints](https://devhints.io/)
-  Rico's cheatsheets are not just focused on the command-line and
-  include a plethora of other cheatsheets related to programming.
+</details>
 
-- [eg](https://github.com/srsudar/eg)
-  provides detailed examples with explanations on the command-line.
-  Examples come from the repository, but `eg` supports displaying
-  custom examples and commands alongside the defaults.
+<details>
+<summary><strong>How is this different from man pages?</strong></summary>
 
-- [kb](https://github.com/gnebbia/kb)
-  is a minimalist command-line knowledge base manager.
-  kb can be used to organize your notes and cheatsheets in a minimalist
-  and clean way. It also supports non-text files.
+- **📚 Man pages**: Comprehensive, technical, reference documentation
+- **⚡ tldr pages**: Practical examples, common use cases, quick reference
 
-- [navi](https://github.com/denisidoro/navi)
-  is an interactive cheatsheet tool, which allows you to browse through
-  specific examples or complete commands on the fly.
+Think of tldr as your friendly cheat sheet, while man pages are the complete manual.
 
-- [bropages (deprecated)](http://bropages.org)
-  are a highly readable supplement to man pages.
-  It shows concise, common-case examples for Unix commands.
-  The examples are submitted by the user base, and can be voted up or down;
-  the best entries are what people see first when they look up a command.
+</details>
 
-## What does "tldr" mean?
+<details>
+<summary><strong>Can I use tldr offline?</strong></summary>
 
-TL;DR stands for "Too Long; Didn't Read".
-It originated as Internet slang, where it is used to indicate that a long text
-(or parts of it) has been skipped as too lengthy.
-Read more in How-To Geek's [article](https://www.howtogeek.com/435266/what-does-tldr-mean-and-how-do-you-use-it/).
+**Yes!** Most clients cache pages locally for offline use. The pages are also available as downloadable PDFs.
+
+</details>
+
+<details>
+<summary><strong>How often are pages updated?</strong></summary>
+
+Pages are continuously updated by our community. The clients typically update their local cache automatically or when you run `tldr --update`.
+
+</details>
+
+---
+
+<div align="center">
+
+### 🎉 Thank you to all our contributors!
+
+[![Contributors](https://contrib.rocks/image?repo=tldr-pages/tldr)](https://github.com/tldr-pages/tldr/graphs/contributors)
+
+---
+
+**💬 Questions? Ideas? Join our community!**
+
+[![Matrix Chat](https://img.shields.io/badge/Chat-Matrix-brightgreen?style=for-the-badge&logo=matrix)](https://matrix.to/#/#tldr-pages:matrix.org)
+[![Mastodon](https://img.shields.io/badge/Follow-Mastodon-6364FF?style=for-the-badge&logo=mastodon&logoColor=fff)](https://fosstodon.org/@tldr_pages)
```

Comment:
Having badges at the top increases visibility of our chatroom and other social's now it will be barely noticed by anyone.